controllers/configmap_controller.go

```diff
@@ -158,6 +162,21 @@ func (r *ConfigMapReconciler) Reconcile(ctx context.Context,
 		return ctrl.Result{}, fmt.Errorf("unable to create signer from private key secret: %w", err)
 	}
 
+  // regenerate the servicesmanifest
+	ign, err := ignition.New(r.client)
+  if err != nil {
+		return ctrl.Result{}, err
+	}
+  argsFromIgnition, err := ign.GetKubeletArgs()
+  if err != nil {
+		return ctrl.Result{}, err
+	}
+  svcData, err := services.GenerateManifest(argsFromIgnition, r.VXLANPort, r.platform, ctrl.Log.V(1).Enabled())
+  if err != nil {
+		return ctrl.Result{}, err
+	}
+  r.servicesManifest = svcData
```

Comment:
This is duplicate code https://github.com/openshift/windows-machine-config-operator/blob/84de66e4398af19af6d60b6608c60a03c8373750/controllers/configmap_controller.go#L99-L111

it should be shared via a common helper function. The reason for this is if we need to update the logic then we don't want to have to do it in multiple places. This leads to code churn and potential errors.